src/site/_includes/components/Instruction.js

```diff
@@ -16,18 +16,21 @@
 
 const {html} = require('common-tags');
 const capitalize = require('../../_filters/capitalize');
+const {i18n, getLocaleFromPath} = require('../../_filters/i18n');
 
 /**
  * A component to help DRY up common lists of instructions.
  * This helps ensure consistency in our docs and makes it easy
  * to respond when Glitch changes their UI.
+ * @this {EleventyPage}
  * @param {string} type The type of instruction to print.
  * @param {string} listStyle The list style to use. Defaults to 'ul'.
  * @return {string} A list of instructions.
  */
-module.exports = (type, listStyle = 'ul') => {
+function Instruction(type, listStyle = 'ul') {
+  const locale = getLocaleFromPath(this.page && this.page.filePathStem);
```

Comment:
I'm kind of wondering if `this.page && this.page.filePathStem` is useful here... I think if `this.page` is undefined it's still going to call the function but may pass in `undefined` or `false` as the argument, right? The function has a default return value so that would be fine, but it does feel a little sketchy to potentially pass an undefined into a function.